sympy/core/tests/test_args.py

```diff
@@ -2994,6 +2994,15 @@ def test_sympy__simplify__hyperexpand__G_Function():
     from sympy.simplify.hyperexpand import G_Function
     assert _test_args(G_Function([2], [1], [], []))
 
+@XFAIL
```

Comment:
I think it's important to get rid of this `@XFAIL`. I will be off for the 1st of May holidays, next week I'll have a better look.